release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1306,6 +1319,21 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[discrete]
+=== Edge computing Technology Preview features
```

Comment:
Looking at the examples in our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#discrete-headings), I think even if a heading is discrete, it needs to have an ID in it:
```suggestion
[discrete]
[id="edge-computing-tp-features_{context}"]
=== Edge computing Technology Preview features
```